ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Thank you for your interest in OpenSALT!
+
+Do the checklist before filing an issue:
+
+- [ ] Is this something you can **debug and fix**? Send a pull request after filing an issue! Bug fixes and documentation fixes are welcome.
+- [ ] Have an idea for a feature? Post the feature request on [Github Issues](http://www.github.com/opensalt/opensalt/Issues).
```

Comment:
Isn't that when this template would be displayed?  We might want to re-phrase.

Reply:
whups... will rephrase